src/github.ts

```diff
@@ -65,16 +65,17 @@ export class OctokitGitHub implements GitHub {
     const options: Octokit.EndpointOptions = {
       owner,
       repo,
-      workflow_id,
-      status: "in_progress"
+      workflow_id
     };
 
     if (branch) {
       options.branch = branch;
     }
 
-    return this.octokit.paginate(
-      this.octokit.actions.listWorkflowRuns.endpoint.merge(options)
-    );
+    return this.octokit
+      .paginate(this.octokit.actions.listWorkflowRuns.endpoint.merge(options))
+      .then(runs =>
+        runs.filter(run => ["in_progress", "queued"].includes(run.status))
```

Comment:
If this run, the one querying for a prior active run, is in progress how would it be possible for a previous run to be queued?

In a build queue it's first in first out so if the previous run was queued that implies the current run has not yet started.